appengine/loopback/server/server.js

```diff
@@ -23,20 +9,21 @@ app.start = function() {
   // start the web server
   return app.listen(function() {
     app.emit('started');
-    console.log('Web server listening at: %s', app.get('url'));
+    var baseUrl = app.get('url').replace(/\/$/, '');
```

Comment:
These files are generated by the loopback app scaffolder. I'm not going to manually modify any of them.